### PR TITLE
Add Goyave

### PR DESCRIPTION
**Pull request by System-Glitch**
Added [Goyave](https://github.com/System-Glitch/goyave/).

Ref issue: [#108](https://github.com/System-Glitch/goyave/issues/108)

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -1177,6 +1181,35 @@ func startWebgo() {
...
+	}
+	r.String(http.StatusOK, "hello")
+}
+
+func getGoyaveRoutes(router *goyave.Router) {
+	router.Get("/hello", goyaveHandler)
+}
+
+func startGoyave() {
+	if err := config.LoadFrom("goyavecfg.json"); err != nil {
```
Comment (smallnest) [inline server.go]: Should it parse a template with args instead of a config file?

Comment (System-Glitch) [reply server.go]: This framework is opinionated and requires a configuration file. (By default it uses `config.json` in the working directory, but you can manually load one too)
However, this config file can also contain environment variables, if you prefer.

Comment (System-Glitch) [reply server.go]: Alternatively to avoid problems with missing files, I can create a temp file and load it.

**Comment by System-Glitch** [comment]
Any news?

**Comment by smallnest** [comment]
Don't use extra file for configuring the web framework. Use variables in server.go to keep consistent with other web frameworks

File: server.go [hunk 2]
```diff
@@ -1177,6 +1182,58 @@ func startWebgo() {
...
+
+func getGoyaveRoutes(router *goyave.Router) {
+	router.Get("/hello", goyaveHandler)
+}
+
+func startGoyave() {
+	cfg := []byte(`
+	{
+		"server": {
+			"host": "127.0.0.1",
```
Comment (smallnest) [inline server.go]: use 0.0.0.0 as host
use port variable as this port
Do not use hardcode because we may test in different server and port

**Pull request merged**